### ISSUE TITLE
WER=100 using pretrained model V2  on Librispeech

### ISSUE THREAD

**Issue by yongfu-li** [SeanNaren/deepspeech.pytorch#442]
To test the modelV1.2, we downloaded pretrained modelV2 and test on LibriSpeech,and got a accuracy of WER =100.000,  CER=82.256. The decoded output is  a blank string.  
This is my command line:
`python test.py --model-path /home/xuxinzi/deepspeech.pytorch-1.2/librispeech_pretrained.pth --test-manifest /home/xuxinzi/deepspeech.pytorch-1.2/data/libri_test_clean_manifest.csv --cuda`
Could you help me to find out the problem?

**Comment by nhh1501** [comment]
I think the problem lies in data_loader.py

**Comment by triwoods** [comment]
@yongfu-li I am getting the same issue, did you find ways to fix it? Thanks!

**Comment by archiki** [comment]
Hi, I made the change as suggested by @nhh1501, but my results are Test Summary 	Average WER 35.678	Average CER 13.472	
Are there other changes to be made as well ?

**Comment by SeanNaren** [comment]
Use the latest pre-trained models release!
